src/cloudflare/internal/ai-api.ts

```diff
@@ -74,6 +82,22 @@ export class AiInternalError extends Error {
   }
 }
 
+async function blobToBase64(blob: Blob): Promise<string> {
+  const arrayBuffer = await blob.arrayBuffer();
```

Comment:
Let's add a todo comment here...
```suggestion
  // TODO(soon): This is better implemented using the node::buffer API
  // but we cannot get to that from here currently. Once the node:buffer
  // API (actually, `node-internal:internal_buffer`) is available to be imported
  // here we should update this code to use it instead.
  const arrayBuffer = await blob.arrayBuffer();
```